pantheon-bundle/src/test/java/com/redhat/pantheon/servlet/ModuleJsonServletTest.java

```diff
@@ -68,22 +68,19 @@ void getQuery() {
     void resourceToMap() throws Exception {
         // Given
         slingContext.create()
-                .resource("/content/repositories/repo/module/en_US/1",
+                .resource("/content/repositories/repo/module/en_US/released",
```

Comment:
Not sure these test updates are necessary - but since you're making them, it might be worth it to use string constants to avoid confusion between "release" vs "released". I noticed in the previous class that you didn't do it and I neglected to mention anything since it was pretty local, but now that I see this, it gives me pause.

Reply:
Yeah, these magic strings are all over the place. I have TODOs to externalize the strings. I can do it as part of this PR.

Reply:
Not sure what you mean by not necessary though... every test method which is fetching the draft or released version of a module specifically will need to be updated.

On second thought about the magic strings: I will externalize the ones in the model objects, but not the tests as I think it reduces their readability, specially since they are using these strings as hard-coded prerequisite data.